run_all_tests.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sbt clean scalastyleAll compile coverage test coverageOff coverageReport dependencyUpdates
```

Comment:
- not working because you dont alias command for scalastyleAll
- not working because you dont have plugin for dependencyUpdates
- your script is called run all testa and you are not running it tests here